### ISSUE TITLE
[BUG] #61936 copy & paste to and from claude code in xterm using the mouse wheel no longer possible 

### ISSUE THREAD

**Issue by yesornobutbetteryes** [anthropics/claude-code#65929]
### Preflight Checklist

- [x] I have searched [existing issues](https://github.com/anthropics/claude-code/issues?q=is%3Aissue%20state%3Aopen%20label%3Abug) and this hasn't been reported yet
- [x] This is a single bug report (please file separate reports for different bugs)
- [x] I am using the latest version of Claude Code

### What's Wrong?

This is an accessibility issue. I am a one-handed user and cannot use the Shift+click workaround.
Related to #61936
mouseTracking: false in settings.json has no effect, CLAUDE_CODE_NO_MOUSE=1 has no effect, downgrading to 2.1.148 has no effect. Please provide a proper way to disable mouse tracking for accessibility reasons.

### What Should Happen?

in a Fedora 42 installation, I was marking text in an editor and did paste it to claude code running in an xterm by pressing the mouse wheel. This did work in the past. In the current version, it is no longer possible.

### Error Messages/Logs

```shell
no error message
```

### Steps to Reproduce

always, mark text, using mouse wheel button to paste it to claude code running in an xterm

### Claude Model

Opus

### Is this a regression?

Yes, this worked in a previous version

### Claude Code Version

2.1.167

### Platform

Anthropic API

### Operating System

Other Linux

### Terminal/Shell

Xterm
